package.json

```diff
@@ -52,7 +52,7 @@
     "open-analyzer": "EXPO_PUBLIC_OPEN_ANALYZER=1 yarn build-web"
   },
   "dependencies": {
-    "@atproto/api": "0.12.29",
+    "@atproto/api": "^0.13.0",
```

Comment:
idk what your thoughts are on pinning the exact version here. We rely so closely on the specific version, I think it'd make sense to pin it

Reply:
oh i didnt even realize we had it pinned before. yea lets.